src/main.cpp

```diff
@@ -35,88 +37,89 @@ constexpr auto VERSION = "[Unkown version]";
 #endif
 
 void printWelcomeMessage() {
-  std::cout << "Touchégg " << VERSION << "." << std::endl;
-  std::cout << "Usage: touchegg [--daemon [start_threshold finish_threshold]] "
+  tlg::info << "Touchégg " << VERSION << "." << std::endl;
```

Comment:
Maybe we should move the welcome message to the arg parser class? As it should be displayed with the --help option.
Probably the #ifdef _VERSION... should go there as well.